pkg/api/apiv1/apiv1.go

```diff
@@ -71,8 +75,11 @@ func (a *api) setup() {
 		r.Get("/runs/{runID}", a.GetFunctionRun)
 		r.Delete("/runs/{runID}", a.CancelFunctionRun)
 		r.Get("/runs/{runID}/jobs", a.GetFunctionRunJobs)
-		// r.Get("/functions", a.GetFunctions)
-		// r.Get("/functions/{id}", a.GetFunction)
+
+		r.Get("/apps/{appName}/functions", a.GetAppFunctions) // Returns an app and all of its functions.
```

Comment:
**suggestion**: Should this be `appID` to match what field is in the response and how it looks like in the Inngest client's id param?
```suggestion
		r.Get("/apps/{appID}/functions", a.GetAppFunctions) // Returns an app and all of its functions.
```

Reply:
These are for internal usage only, and I prefer `appName` because ID is ambiguous.